client/src/pages/Courses/Courses.js

```diff
@@ -17,11 +19,17 @@ const Courses = () => {
     const [isEditing, setIsEditing] = useState(false);
 
     const { data, columns, createCourse, updateCourse, deleteCourse, createCoursesFromUniApi, prefillCourse } = useCourses();
+    const history = useHistory();
+    const { courseId } = useParams();
 
     useEffect(() => {
         setSelectedCourse(isAddingCourse ? {} : data[currRow] || {});
         setIsEditing(false);
-    }, [data, currRow, isAddingCourse]);
+        const rowID = data.findIndex((course) => course._id === courseId);
+        setCurrRow(rowID.toString());
+        setIsAddingCourse(courseId === "new");
+        console.log(rowID);
```

Comment:
pls remove : )

Reply:
my bad. thought the checker had a console log checker